ci/appveyor/linux_prepare.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Could you please elaborate further why you need so many hacks during the install here?

While [the script for CircleCI](https://github.com/aeternity/aeternity/blob/master/.circleci/config.yml#L58-L64) is about Ubuntu 16.04, it's straight forward.

And concretely why:
- `sudo apt remove -y erlang-base` ?
- `sudo apt --fix-broken install -y` ?
- this additional dependancy `libsctp1` ?
- the specific 0v5 in `libwxbase3.0-0v5 libwxgtk3.0-0v5` ?
- `libwxbase3.0-0v5 libwxgtk3.0-0v5` is not installed at once with the other deps above ?